src/database.py

```diff
@@ -6,4 +6,11 @@
 
 
 def init_db():
+    """
+    Initializes the database connection.
+
+    This function should be called once at the start of the application and will create the
+    database and collections if they don't exist.
+    """
+    print(db.list_collection_names())
```

Comment:
not sure if initializing the client above does what this function is meant to do, so we can probably just delete it